app/public/presentation/common/courses/course.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, Input } from '@angular/core';
-import { Course } from './view-model/course.view-model';
+import { Course } from '../../../domain/entities/course.entity';
```

Comment:
I'm having some problems to implement this. The baseUrl serves only for compiling. The compiled code still has the 'require("api/api.router.js")', then the code fails when running on node. I found some links related to this:
https://github.com/Microsoft/TypeScript/issues/5304
http://stackoverflow.com/questions/41246294/using-typescripts-baseurl-compiler-option-with-node

What do you think?